docs/build/tutorials/platform/deploying-cross-chain-evm-bridge.md

```diff
@@ -0,0 +1,324 @@
+
+Let's make a new directory `deploy-bridge`, where we will be keeping our bridge codes. We will be using the following repositories -
+
+* [ChainSafe/chainbridge-deploy](https://github.com/ChainSafe/chainbridge-deploy) - This will help us in setting up of our bridge contracts
+* [ChainSafe/chainbridge](https://github.com/ChainSafe/ChainBridge) - This will help us in setting up of our off-chain relayer.
+
+### Installing Chainbridge Command Line Tool
+
+Using the following command, we can clone and install Chainbridge's command-line tool. This will help us in setting up bridge contracts and demonstrating bridge transfers. Once the bridge contracts are deployed, you can use its ABI and contract address to set up your UI.
+
+```bash
+git clone -b v1.0.0 --depth 1 https://github.com/ChainSafe/chainbridge-deploy \
+&& cd chainbridge-deploy/cb-sol-cli \
+&& npm install \
+&& make install
+```
+
+This will build the contracts and installs the `cb-sol-cli` command.
+
+### Setting Up Environment Variables
+
+Let's set up environment variables, so that, we do not need to write their values every time we issue a command. Move back to the `deploy-bridge` directory (main project directory) and make a new file `configVars`. Put the following contents inside it -
+
+```env
+SRC_GATEWAY=https://subnets.avax.network/wagmi/wagmi-chain-testnet/rpc
+DST_GATEWAY=https://api.avax-test.network/ext/bc/C/rpc
+
+SRC_ADDR="<Your address on WAGMI>"
+SRC_PK="<your private key on WAGMI>"
+DST_ADDR="<Your address on Fuji>"
+DST_PK="<your private key on Fuji>"
+
+SRC_TOKEN="0x3Ee7094DADda15810F191DD6AcF7E4FFa37571e4"
+RESOURCE_ID="0x00"
+```
+
+* `SRC_ADDR` and `DST_ADDR` are the addresses that will deploy bridge contracts and will act as a relayer.
+* `SRC_TOKEN` is the token that we want to bridge. Here is the address of the wrapped ERC20 version of the WGM coin aka wWGM.
+* `RESOURCE_ID` could be anything. It identifies our bridged ERC20 tokens on both sides (WAGMI and Fuji).
+
+Every time we make changes to these config variables, we have to update our bash environment. Run the following command according to the relative location of the file. These variables are temporary and are only there in the current terminal session, and will be flushed, once the session is over. Make sure to load these environment variables anywhere you will using them in the bash commands (like `$SRC_GATEWAY` or `$SRC_ADDR`)
+
+```bash
+source ./configVars
+```
+
+## Setting Up Source Chain
+
+We need to set up our source chain as follows -
+
+* Deploy Bridge and Handler contract with `$SRC_ADDR` as default and only relayer
+* Register the `wWGM` token as a resource on the bridge
+
+### Deploy contracts
+
+The command-line tool `cb-sol-cli` will help us to deploy the contracts. Run the following command in the terminal session where the config vars are loaded. It will add `SRC_ADDR` as the default relayer for relaying events from the Wagmi chain (source) to the Fuji chain (destination).
+
+```bash
```

Comment:
You should also note in **BOLD** that you should choose a very large expiry value. Expired proposals were the only time manual intervention was required on the bridge. Once a proposal expiry config is set, it cannot be modified.